### ISSUE TITLE
No longer works with org.webjars.bowergithub.* packages

### ISSUE THREAD

**Issue by kanwei** [weavejester/ring-webjars#7]
Apparently webjars is moving over to org.webjars.bowergithub.* packages instead of org.webjars.bower* packages. When doing it via bowergithub, the version is no longer added to the path, so it goes from:

``` "META-INF/resources/webjars/jquery/3.3.1/src/var/support.js"```
to
``` "META-INF/resources/webjars/jquery/src/var/support.js"```

Because the ring-webjars ```webjars-pattern``` depends on a version, org.webjars.bowergithub.* packages cannot be used.

**Comment by olimsaidov** [comment]
This workaround works:

```
(defn wrap-webjars
  [handler prefix]
  (with-redefs [ring.middleware.webjars/webjars-pattern #"META-INF/resources/webjars/([^/]+)/(?:([^/]+)/)?(.*)"]
    (ring.middleware.webjars/wrap-webjars handler prefix)))
```

**Comment by weavejester** [comment]
Should be fixed by: 117770f21d47a365aa27e7f2deebea9d56b30e9f
